sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -586,3 +510,157 @@ def _reset_extra_properties_in_genbank(self, obj: sbol3.Component, seq_rec: SeqR
         # TODO: temporalily hardcoding version as "1"
         seq_rec.annotations["sequence_version"] = self.DEFAULT_GB_SEQ_VERSION
 
+    
+    def _handle_features_gb_to_sbol(self, record: SeqRecord, comp: sbol3.Component, seq: sbol3.Sequence) -> None:
+        """Helper function for setting sequence features and their qualifiers to SBOL,
+        by using a modified, extended SBOL3 Sequence Feature class - Feature_GenBank_Extension.
+        :param record: GenBank SeqRecord instance for the record which contains sequnce features
+        :param comp: Instance of the SBOL3 Component 
+        :param seq: The Sequence used in the GenBank record corresponding to sbol3 comp
+        """
+        # parse if genbank record has any features
+        if record.features:
```

Comment:
Note: I am not asking for a change, I am making an observation.

After reviewing the code for the case where there are `record.features` I scrolled down to see what happens when there are no record features. I think the answer is that nothing happens, but it was hard to tell. A different option is to short-circuit the method and de-indent the body:

```python
if not record.features:
    # There is nothing to do if there are no features
    return
comp.features = []
for ind, gb_feat in ....
    ...
# rest of method goes here
```

Reply:
@tcmitchell Thanks for pointing this out! I have inculcated this.